File: chapters/lab_1/scripts/RadiologyLabAnalysis.py
import pandas as pd
import numpy as np, scipy.stats as st

lab_data = pd.read_table("radiology_lab.dat", header=0, names=[
  'Scenario', 'Replication', 'TimeInSystem'
]).dropna()

print(lab_data.head())

conf_int = lab_data['TimeInSystem'].agg({
  'Mean': 'mean',
  'CI Half Width': lambda x: x.sem() * st.norm.ppf(0.975),
})

print(conf_int)

File: chapters/lab_1/scripts/RadiologyLabAnalysis.R
require(dplyr)
setwd("H:/ENGSCI355/labs/RC1")
RadiologyLabdata = read.table("radiology_lab.dat", sep="\t",
                      col.names=c("Scenario", "Replication", "TimeInSystem"),
                      skip=1)
RadiologyLabdata = na.omit(RadiologyLabdata)
head(RadiologyLabdata)

RadiologyLabdata %>%
  summarise(lower95CI=t.test(TimeInSystem)$conf.int[1], mean=mean(TimeInSystem),
            upper95CI=t.test(TimeInSystem)$conf.int[2])
